contracts/Cruize.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+import "./base/Proxy.sol";
+import "./base/AaveV2Wrapper.sol";
+
+/**
+ * @author CRUIZE.
+ * @title Cruize AssetPool.
+ */
+contract Cruize is Proxy, AaveV2Wrapper {
+    receive() external payable {}
+
+    //----------------------------//
+    //     Mutation Functions     //
+    //----------------------------//
+
+    /**
+     * @notice initialize will initialize the crContract.
+     * @param _cruize_wallet address -  Cruize Wallet address.
+     * @param _crContract address -  ERC20Upgradeable Contract.
+     */
+
+    function initialize(address _cruize_wallet, address _crContract)
```

Comment:
it's an external function and the _cruize_wallet   this type of parameter name is used for. The internal function can. can you change it?